app/models/manageiq/providers/inventory.rb

```diff
@@ -57,5 +44,73 @@ def parse
     def inventory_collections
       parse.inventory_collections
     end
+
+    # Based on the given provider/manager class, this returns correct persister class
+    #
+    # @param klass class of the Provider/Manager
+    # @return [Class] Correct class name of the persister
+    def self.persister_class_for(klass)
```

Comment:
@agrare I only changed order of these methods in file, no change there

Reply:
I'm good with this for now, I think these should all be private and then all of the yardoc comments are unnecessary

Reply:
Actually where is persister_class_for used?

Reply:
it's used by old refresher: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/base_manager/refresher.rb#L168

Reply:
Ah okay :+1: